app/konnect/configure/runtime-manager/runtime-groups/declarative-config.md

```diff
@@ -133,10 +153,9 @@ the example with your generated UUID:
     ```sh
     creating service 9595B5F9-3B6A-4C48-BE93-9EC1B0EA487A
     creating route mockpath
-    creating service-package MyService
-    creating service-version 1
+    creating service MyService
```

Comment:
Another leftover from ServiceHub config. This should only be 2 items:

```
creating route mockpath
creating service MyService
```
Note to self to fix.